src/wp-includes/general-template.php

```diff
@@ -3326,6 +3326,10 @@ function feed_links_extra( $args = array() ) {
 	 */
 	$args = apply_filters( 'feed_links_extra_args', $args );
 
+	/*
+	 * Use the queried object rather than relying on the global $post.
+	 * feed_links_extra() should not depend on $GLOBALS['post'] being set. See #63263
```

Comment:
```suggestion
	 * The template conditionals are referring to the global query, so the queried object is used rather than
	 * depending on a global $post being set.
```